.github/workflows/ci_docs.yml

```diff
@@ -137,7 +140,18 @@ jobs:
 
       # Build the documentation
       - name: Build the documentation
-        run: make -C doc clean all
+        run: make -C doc clean html
+
+      - name: Build the PDF
+        run:  make -C doc pdf
+
+      # Temporarily upload the built PDF to the artifacts
+      - name: Upload PDF to artifacts
+        uses: actions/upload-artifact@v4.5.0
+        with:
+          name: pygmt-docs-pdf
+          path: doc/_build/latex/pygmt.pdf
+        if: matrix.os == 'ubuntu-latest'
```

Comment:
```suggestion
```

Reply:
Actually we still need to upload the PDF to artifacts, so that we can deploy it to the gh-pages later. So let's keep this step.

Need to wait for https://github.com/GenericMappingTools/pygmt/pull/3767.